### ISSUE TITLE
Represent folders in UI

### ISSUE THREAD

**Issue by totti4ever** [LibrePhotos/librephotos#129]
Hey, I found it super helpful and in certain use cases super useful when the pictures were also shown in their hierarchical folder structure from the file system or at least there were folder albums, where each folder(with pictures in it) was shown as album.

This is partly a plausibility thing, but also just structure as I have them sorted in a helpful folder hierarchy, which I don't want to manually rebuild based on albums.

I guess, one could simply add an entry on albums, where there is face, things, places already today. Just add folder there?!

**Comment by derneuere** [comment]
I would, I think prioritize reading folders as albums as a scanning option at the first 👍

**Comment by totti4ever** [comment]
But would that mean, you couldn't distinguish an album derived from folders versus one created manually?

**Comment by darrepac** [comment]
Sounds to me a must to have a way to show "Disk album", I mean the one following your folder structure.
I don't know if it has to be part of "My albums" or with a new menu entry "Disk album":
![image](https://user-images.githubusercontent.com/7128994/111042807-770bcc00-843f-11eb-92f6-1246283d51fd.png)

**Comment by MultiCoreNop** [comment]
I would prefer to have this as a separate entry like in darrepac's comment. My directory structure is basically album oriented. Having both entries ("My Albums" and "Disk Albums") would allow to either share the directory or to create a "virtual album" consisting of inter-album images and to share that (PhotoPrism works in a similar way, you can create arbitrary albums or just browse your directory structure as is).

**Comment by LimpingNinja** [comment]
I agree with MultiCoreNop here -- the physical representation should either create virtual albums that represent a disk structure or allow browsing via disk structure. I have my albums sorted by Trip already (thanks to Elodie) and like the layout.

**Comment by TheAnachronism** [comment]
So, is something going on here, or does this just have no prio?
Why do so many photo solutions not support existing library structure?
Especially annoying if one already has an organized library of thousands of photos -.-

**Comment by davi024** [comment]
We need this option to maintain our folder structure and to be able to grow it with Librephotos.

My idea is to have all the photos synchronized in a root folder and to be able to generate the albums in my current structure, it would be nice to do it with inodes so that I don't have to duplicate a file if I want to have it in several albums.
Also that Librephotos visualizes my current structure to label the photographs and use facial recognition.

**Comment by nagarrido** [comment]
Hi, is there any update on this feature?

**Comment by sterys** [comment]
My vote would be to calle it Disk Folders rather then Disk Album since that it was it is. 
 
This should make it more clear that it just follows the disk layout and that the only way to change it is to move around the files and folders externally and then do a rescan. 

It would be okay if you could not do everything with an folder as you can with an Album. I would like to be able to select all files in an folder and create an manual Album thought.

**Comment by nagarrido** [comment]
Maybe there could be both options. To see folders separate from albums, and to generate an album linked to a specific folder?

**Comment by mw911** [comment]
Hi, I would like to self host my photos and LibrePhotos is my favourite of all solutions tested so far. Only thing I am also missing is a hierarchical folder structure view. I think many users organise their files similar to year/event

**Comment by 1michaelbrown** [comment]
Is there an update on this. I would love a directory first approach I think most people that store photos professional would too

**Comment by antoniocaruso** [comment]
I suggest to view the way Lightroom organize photos, it is one of the most important software for professional people. Especially browsing the metadata of large collection is very efficient.

**Comment by derneuere** [comment]
There is now a folder view :)
